source/quick-start/create-a-connection-string.txt

```diff
@@ -24,34 +24,32 @@ To connect to an instance or deployment not hosted on Atlas, see
       To retrieve your connection string for the deployment that
       you created in the :ref:`previous step <node-quick-start-create-deployment>`,
       log into your Atlas account and navigate to the
-      :guilabel:`Database` section and click the :guilabel:`Connect` button
+      :guilabel:`Clusters` section and click the :guilabel:`Connect` button
       for your new deployment.
 
-      .. figure:: /includes/figures/atlas_connection_select_cluster.png
+      .. figure:: /includes/figures/atlas_connect.png
          :alt: The connect button in the clusters section of the Atlas UI
 
-      Proceed to the :guilabel:`Connect your application` section and select
-      "Node.js" from the :guilabel:`Driver` selection menu and the version
+      Select the :guilabel:`Drivers` option from the :guilabel:`Connect to your application` 
+      section. Once you are in the :guilabel:`Connecting with MongoDB Driver` 
```

Comment:
[Style Guide](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-after) - Use "once" only for specifying doing something one time
```suggestion
      section. Then in the :guilabel:`Connecting with MongoDB Driver` 
```